app/api/v1/redflags/views.py

```diff
@@ -0,0 +1,25 @@
+from flask_restful import Resource
+from flask import request, make_response, jsonify
+import json
+
```

Comment:
```suggestion
```
This is an unnecessary blank line. Configure linting to help you keep track of standards

Reply:
You still haven't removed the extra blank line here